tools/jnimarshalmethod-gen/App.cs

```diff
@@ -8,32 +7,73 @@
 
 using Java.Interop;
 
-namespace Xamarin.Android.Tools.JniMarshalMethodGenerator {
+using Mono.Cecil;
+using Mono.Options;
+using Mono.Collections.Generic;
+using Java.Interop.Tools.Cecil;
 
-	class App {
+namespace Xamarin.Android.Tools.JniMarshalMethodGenerator {
```

Comment:
The movement of "this" line is weird, and I *suspect* it's because of two lines down, where `class App {` became `class App`.